ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetPrecompiledContractRegistries.java

```diff
@@ -60,6 +61,13 @@ public static PrecompileContractRegistry bls12(
     return registry;
   }
 
+  public static PrecompileContractRegistry cancun(
```

Comment:
this makes sure that this precompile is only seens when Cancun activates, right?